src/olympia/search/tests/test_views.py

```diff
@@ -671,15 +671,19 @@ def test_find_addon_default_non_en_us(self):
                 assert addon_en.pk in result
                 assert addon.pk in result
 
-                response = self.client.get(url, {'q': 'Banana'})
+                # This will always match regardless of language, because it's
+                # part of the description in the default locale.
+                response = self.client.get(url, {'q': 'Browser'})
```

Comment:
no haz 🍌 ? 😿

Reply:
why _doesn't_ `Banana` work anymore?  I don't see any other changes to this file.

Reply:
`Banana` does work everywhere but it was a poor example, because it's present in every single translation in one form or another. `Browser` is a better example because it's *only* in the description in the default locale.

However, `plátanos` below no longer works in every locale, because it's only part of the Spanish translation and Spanish is not the default locale for this add-on. So it will only work when the client is searching in Spanish.